### PR TITLE
[[ Documentation ]] Changes to miterLimit.lcdoc

### PR DESCRIPTION
**Pull request by asayd**
- Added missing references and links.

### CODE REVIEW

File: docs/dictionary/property/miterLimit.lcdoc [hunk 1]
```diff
@@ -21,25 +21,27 @@ set the miterLimit of graphic "polygon 1" of this card to 5
...
-<lineSize> of the graphic.
-
->*Note:* This only affects polygon, freehand curve and line graphics
-> which have their antialiased set to true.
-
-References: roundEnds (property), lineSize (property),
-antialiased (property)
+Set the <miterLimit> property to specify how far miter <joinStyle|joins>
+can extend from the meeting point of two edges before a bevel 
+<joinStyle|joins> is used.
```
Comment (montegoulding) [inline docs/dictionary/property/miterLimit.lcdoc]: @asayd should be join here rather than joins

File: docs/dictionary/property/miterLimit.lcdoc [hunk 2]
```diff
@@ -21,25 +21,27 @@ set the miterLimit of graphic "polygon 1" of this card to 5
...
+
+The <miterLimit> is used to calculate the maximum size of a miter 
+<joinStyle|joins> before it is rendered as a bevel. The size of a 
+miter <joinStyle|joins> is the length in pixels from the corner to the
```
Comment (montegoulding) [inline docs/dictionary/property/miterLimit.lcdoc]: here too and the line above

**Comment by asayd** [comment]
@montegoulding Good catch. Perils of copy/paste!

**Comment by BerndN** [comment]
@asayd Hi Devin, just trying to understand how to fix dictionary entries I noticed in your last correction on line 37 a remaining "joins"
Maybe just fix it so it can be reviewed and merged
Thanks for all the work you are doing on the dictionary.

**Comment by asayd** [comment]
@BerndN Thanks for catching that. Clearly I was staring at it too long! :)

**Comment by montegoulding** [comment]
@livecode-vulcan review ok d9e550a

**Comment by livecode-vulcan** [comment]
💙 review by @montegoulding ok d9e550a1d2991a1581e5a4848f02f362018b9eb5

**Comment by livecode-vulcan** [comment]
😞 test failure d9e550a1d2991a1581e5a4848f02f362018b9eb5
- try-community-linux-x86: [pending](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/505)
- try-community-mac: [pending](https://vulcan.livecode.com/builders/try-community-mac/builds/510)
- try-community-linux-x86_64: [pending](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/505)
- try-community-emscripten: [failure](https://vulcan.livecode.com/builders/try-community-emscripten/builds/507)
- try-community-ios: [pending](https://vulcan.livecode.com/builders/try-community-ios/builds/512)
- try-community-win-x86: [pending](https://vulcan.livecode.com/builders/try-community-win-x86/builds/505)
- try-community-android-armv6: [pending](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/506)

**Comment by montegoulding** [comment]
@livecode-vulcan try d9e550a

**Comment by livecode-vulcan** [comment]
😎 test success d9e550a1d2991a1581e5a4848f02f362018b9eb5
- try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/547)
- try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/552)
- try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/547)
- try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/551)
- try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/556)
- try-community-win-x86: [success](https://vulcan.livecode.com/builders/try-community-win-x86/builds/548)
- try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/548)

**Pull request merged**